code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -83,6 +83,67 @@
 			new /obj/item/reagent_containers/glass/waterbottle/relic(src)
 		if(30)
 			new /obj/item/reagent_containers/glass/bottle/necropolis_seed(src)
+		if(31)
+			if(prob(30))
+				new /obj/item/power_tube/filled(src)
+			else
+				new /obj/item/power_tube(src)
+
+//Golden Amulet
+/obj/item/clothing/neck/necklace/golden_amulet
+	name = "golden amulet"
+	desc = "A very old necklace, made out of gold. It has a bloody-red gem inserted into it."
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+	icon_state = "golden_amulet"
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	var/used = FALSE
+
+/obj/item/clothing/neck/necklace/golden_amulet/equipped(var/mob/user, var/slot)
+	. = ..()
+	if(used)
+		unequip(user)
+		return
+	if(slot == ITEM_SLOT_NECK)
+		RegisterSignal(user, COMSIG_MOB_DEATH, .proc/revival)
+	else
+		unequip(user)
+
+/obj/item/clothing/neck/necklace/golden_amulet/dropped(var/mob/user)
+	. = ..()
+	unequip(user)
+
+/obj/item/clothing/neck/necklace/golden_amulet/proc/unequip(var/mob/user)
+	UnregisterSignal(user, COMSIG_MOB_DEATH)
+
+/obj/item/clothing/neck/necklace/golden_amulet/proc/revival()
+	if(used)
+		return
+	var/mob/living/user = loc
+	if(!istype(user))
+		return
+	user.revive(full_heal = 1, admin_revive = 1)
```

Comment:
NO. GOD, PLEASE NO.

Reply:
ITS A ONE USE FUCKING SUPERDRAKE DROP WHAT DO YOU WANT